src/dsp_tools/commands/excel2json/resources.py

```diff
@@ -29,76 +29,120 @@
 languages = ["en", "de", "fr", "it", "rm"]
 
 
```

Comment:
No content was changed, the order of the functions now follow the flow of the main function better.